### PR TITLE
Fix #624

### LINKED ISSUE #624: GetUtils::capitalize method!
```
static String capitalize(String s, {bool firstOnly = false}) {
    if (isNullOrBlank(s)) return null;
    if (firstOnly) return capitalizeFirst(s);

    var lst = s.split(' ');
    var newStr = '';
    for (var s in lst) {
      newStr += capitalizeFirst(s);
    }
    return newStr;
  }
```

This method omits spaces between words once capitalized.
I think below function block is right one.

```
static String capitalize(String s, {bool firstOnly = false}) {
  if (isNullOrBlank(s)) return null;
  if (firstOnly) return capitalizeFirst(s);
  return s.split(' ').map(capitalizeFirst).join(' ');
}
```
 
Thank you.

### PR DESCRIPTION
**Pull request by zl910627**
closes #624

### CODE REVIEW

**Review by Nipodemos — APPROVED** [review]
Tests passed, LGTM!
Now is just wait for @jonataslaw approval

**Comment by jonataslaw** [comment]
Thanks for your contribution
LGTM, I'll merge

**Pull request merged**
